### PR TITLE
Create product.html

### PR DESCRIPTION
**Pull request by suhailkakar**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [X] 🚩 Other

## Description
Created a new component 

## Added to documentation?

- [ ] 📜 readme
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: components/product.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="min-h-screen bg-gray-100 flex flex-col justify-center">
+  <div class="relative m-3 flex flex-wrap mx-auto justify-center">
+    <div class="relative max-w-sm min-w-[400px] bg-white shadow-md rounded-3xl p-2 mx-1 my-3 cursor-pointer">
+      <div class="overflow-x-hidden rounded-2xl relative">
+        <img class="h-70 rounded-2xl w-full object-cover" src="https://unblast.com/wp-content/uploads/2020/10/Shoes-Apparel-Mockup.jpg">
```
Comment (thinkverse) [inline components/product.html]: Please change this to an image from a service like Unsplash instead. 👍

File: components/product.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <p class="text-lg font-semibold text-gray-900 mb-0">PUMA Men's Sneakers</p>
+          <p class="text-md text-gray-800 mt-0">$70.50</p>
+        </div>
+         <div class="mt-2">
+             <div class="mt-4 pl-2 mb-2 flex justify-between ">
+             <div class="bg-red-500 mr-2 w-5 h-5 rounded-full mt-0"></div>
+             <div class="bg-green-500 mr-2 w-5 h-5 rounded-full mt-0"></div>
+             <div class="bg-yellow-500 w-5 h-5 rounded-full mt-0"></div>
+        </div>
+         </div>
```
Comment (thinkverse) [inline components/product.html]: If you could make sure the code is properly indented that would be awesome. 🤘

Comment (suhailkakar) [reply components/product.html]: Updated :)

File: components/product.html (hunk 1 above)
Comment (suhailkakar) [reply components/product.html]: Thanks, Updated :)

File: components/product.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  <div class="relative m-3 flex flex-wrap mx-auto justify-center">
+    <div class="relative max-w-sm min-w-[400px] bg-white shadow-md rounded-3xl p-2 mx-1 my-3 cursor-pointer">
+      <div class="overflow-x-hidden rounded-2xl relative">
+        <img class="h-70 rounded-2xl w-full object-cover" src="https://images.unsplash.com/photo-1575537302964-96cd47c06b1b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80" />
+        <p class="absolute right-2 top-2 bg-white rounded-full p-2 cursor-pointer group">
+          <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 group-hover:opacity-70" fill="none" viewBox="0 0 24 24" stroke="gray">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z" />
+          </svg>
+        </p>
+      </div>
```
Comment (thinkverse) [inline components/product.html]: This section has an overflow issue where the image section is scrollable to reveal the heart icon.

https://user-images.githubusercontent.com/2221746/135605133-7a3a19f2-39ab-4240-abc6-995bdfdf4221.mov

**Pull request merged**